src/tailwind/defaultConfig.ts

```diff
@@ -56,7 +60,8 @@ export default {
       "high-deal": "#E34A15",
       "fair-deal": "#EAA403",
       "good-deal": "#81B040",
-      "great-deal": "#149246",
+      "great-deal": colors.darkGreen,
+      "green-dark": colors.darkGreen,
```

Comment:
Colour is reused but has a very specific name so I decide to add an export with a more generic name and keep backwards compatibility.

Reply:
```suggestion
      "green-dark": colors.greenDark,
```

for the sake of consistency

Reply:
consider asking a designer what they call the color in the styleguide

Reply:
Not available based on what I found on my own I think they reused the existing one made the same suggestion to Enrique to add it with the name `green-dark` to the additional colors section https://www.figma.com/file/Hg2Y3FaaStmRnswYAL9PY0/Core-UI-Components?node-id=0%3A1